repose-aggregator/core/repose-core-api/src/main/resources/META-INF/schema/container/container-configuration.xsd

```diff
@@ -252,6 +257,55 @@
         </xs:simpleContent>
     </xs:complexType>
 
+    <xs:complexType name="ViaConfiguration">
+        <xs:annotation>
+            <xs:documentation>
+                <html:p>
+                    Contains the information needed to configure Repose's Via header.
+                </html:p>
+            </xs:documentation>
+        </xs:annotation>
+        <xs:simpleContent>
+            <xs:extension base="papic:EmptyString">
+                <xs:attribute name="enabled" type="xs:boolean" use="optional" default="true">
+                    <xs:annotation>
+                        <xs:documentation>
+                            <html:p>Indicates whether or not the Via header is created.</html:p>
+                        </xs:documentation>
+                    </xs:annotation>
+                </xs:attribute>
+                <xs:attribute name="content" type="xs:string" use="optional">
```

Comment:
If we want to change the names/intent of the values to be as you described, then that works too. So in that way, the `boolean` attribute would enable/disable the Repose value only and the content would still be user portion only.

content | `boolean` | result
--------|-----------|-------
Stuff | T | _Stuff (Repose/v0.0.0.0)_
Stuff | F | _Stuff_
**Undefined** | T | _(Repose/v0.0.0.0)_
**Undefined** | F | **<NO `Via` HEADER>**